pip/basecommand.py

```diff
@@ -105,6 +105,12 @@ def parse_args(self, args):
     def main(self, args):
         options, args = self.parse_args(args)
 
+        if options.quiet and options.verbose:
```

Comment:
Agreed. You'd also need to then pass the "net verbosity" to then list command below. Although at that point I wonder whether triggering the extra output off the ```-v``` option makes sense - should ```pip list -v -q``` add the extra info or not? Maybe it would be better to have an explicit option for that information.